source/AI.h

```diff
@@ -18,6 +18,7 @@ this program. If not, see <https://www.gnu.org/licenses/>.
 
 #include "Command.h"
 #include "FireCommand.h"
+#include "Hardpoint.h"
```

Comment:
You should be able to forward declare this in here, instead of including the whole header file.